events/amta2022.md

```diff
@@ -34,6 +34,19 @@ Students interested in MT and computational linguistics will be able to connect
 
 ## Workshops
 
+The First Workshop on Corpus Generation and Corpus Augmentation for Machine Translation (CoCo4MT) will be co-located with AMTA 2022 in Orlando, Florida, USA, on September 16th, 2022. 
```

Comment:
To follow our [Style Guide](http://machinetranslate.org/style) and the structure of other articles, I would apply minor stylistic changes:

> The first workshop on Corpus Generation and Corpus Augmentation for Machine Translation (**CoCo4MT**) will be co-located with AMTA 2022 in Orlando, Florida, on 16 September, 2022.

Reply:
Sounds good!